test/new-e2e/pkg/runner/profile.go

```diff
@@ -98,6 +99,24 @@ func newProfile(projectName string, environments []string, store parameters.Stor
 	return p
 }
 
+func defaultEnvironments(environments string, defaultsEnvironments map[string]string) string {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Having `default` in both arguments and function can be confusing, I suggest a renaming and adding a comment to explain what this function does
```suggestion
// mergeEnvironments returns a string with a space separated list of available environments. It merges environments with a `defaultEnvironments` map
func mergeEnvironments(environments string, defaultsEnvironments map[string]string) string {
```